Mathlib/CategoryTheory/Category/Preorder.lean

```diff
@@ -171,28 +168,66 @@ def OrderIso.equivalence (e : X ≃o Y) : X ≌ Y where
 
 end
 
-namespace CategoryTheory
-
 section Preorder
 
 variable {X : Type u} {Y : Type v} [Preorder X] [Preorder Y]
 
-/-- A functor between preorder categories is monotone.
--/
+namespace CategoryTheory.Functor
+
+/-- A functor between preorder categories is monotone. -/
 @[mono]
-theorem Functor.monotone (f : X ⥤ Y) : Monotone f.obj := fun _ _ hxy => (f.map hxy.hom).le
+theorem monotone (f : X ⥤ Y) : Monotone f.obj := fun _ _ hxy => (f.map hxy.hom).le
+
+/-- A functor `X ⥤ Y` between preorder categories as an `OrderHom`. -/
+@[simps!]
+def toOrderHom (F : X ⥤ Y) : (X →o Y) where
+  toFun := F.obj
+  monotone' := F.monotone
+
+end CategoryTheory.Functor
+
+namespace OrderHom
+
+open CategoryTheory
+
+/-- An `OrderHom` as a functor `X ⥤ Y` between preorder categories. -/
+abbrev toFunctor (f : X →o Y) : X ⥤ Y := f.monotone.functor
+
+/-- The equivalence between `X →o Y` and the type of functors `X ⥤ Y` between preorder categories
+`X` and `Y`. -/
+@[simps]
+def equivFunctor : (X →o Y) ≃ (X ⥤ Y) where
+  toFun := toFunctor
+  invFun F := F.toOrderHom
+
+/-- The categorical equivalence beween the category of monotone functions `X →o Y` and the category
+of functors `X ⥤ Y`, where `X` and `Y` are preorder categories. -/
+@[simps! functor_obj_obj inverse_obj unitIso_hom_app unitIso_inv_app counitIso_inv_app_app
+  counitIso_hom_app_app]
+def equivalence : (X →o Y) ≌ (X ⥤ Y) where
```

Comment:
My last nitpick is about the name of this equivalence. The full name here is `OrderHom.equivalence` which is rather generic. I think the name should contain "functor" in a way or an other. Perhaps something like `equivalenceFunctor` is better?

Reply:
Sounds good to me; changed. :) (This name was based on looking at the naming for a few other equivalences in Mathlib, which also seem to indicate only one side of the equivalence in their name. I'm happy to make it more explicit!)